README.md

```diff
@@ -1,14 +1,66 @@
-Fluent UI Apple is a repository containing native UIKit and AppKit controls aligned with Microsoft's Fluent UI design language. 
+# Fluent UI Apple
+Fluent UI Apple is a library containing native UIKit and AppKit controls aligned with [Microsoft's Fluent UI design system](https://www.microsoft.com/design/fluent/#/). 
 
-![CI](https://github.com/microsoft/fluentui-apple/workflows/CI/badge.svg?branch=master)
+![Build Status](https://github.com/microsoft/fluentui-apple/workflows/CI/badge.svg?branch=master)
+![License](https://img.shields.io/github/license/Microsoft/fluentui-apple)
+[![Carthage Compatible](https://img.shields.io/badge/Carthage-compatible-4BC51D.svg?style=flat)](https://github.com/Carthage/Carthage)
+[![CocoaPods Compatible](https://img.shields.io/cocoapods/v/MicrosoftFluentUI)](https://cocoapods.org/pods/MicrosoftFluentUI)
+![Platform](https://img.shields.io/cocoapods/p/MicrosoftFluentUI.svg?style=flat)
 
 ## Getting Started
-To get started developing in this repository for iOS, please check out [the iOS readme file](ios/README.md).
+### Install and use FluentUI
 
-To get started developing in this repository for macOS, please check out [the macOS readme file](macos/README.md).
+#### Requirements
 
-# Contributing
+- iOS 11+ or macOS 10.13+
+- Xcode 11+
+- Swift 5.0+
+
+#### Using Carthage
+
+To integrate FluentUI using Carthage, specify it in your Cartfile:
+
+```
+github "Microsoft/fluentui-apple" ~> X.X.X
+```
+
+#### Using CocoaPods
+
+To get set up with CocoaPods visit their [getting started guide](https://guides.cocoapods.org/using/getting-started.html).
+
+To integrate FluentUI into your Xcode project using CocoaPods, specify it in your Podfile:
+```ruby
+pod 'MicrosoftFluentUI', '~> X.X.X'
+```
+
+then follow the Carthage [integration steps](https://github.com/Carthage/Carthage#adding-frameworks-to-an-application) to add the `FluentUI.framework` into your Xcode project
+
+#### Manual installation
+
+- Download the latest changes from the [FluentUI for Apple](https://github.com/microsoft/fluentui-apple) repository.
+- Move the `fluentui-apple` folder into your project folder.
+- Move the relevant `FluentUI.xcodeproj` into your Xcode project depending on which platform you want to support.
+- In Xcode select your project -> your target -> General -> Embedded Binaries -> add `FluentUI.framework`.
+
+#### Swift Package Manager
+As of this writing, the version of Swift Package Manager shipped with the latest Xcode does not support packages that require resource bundles. As Fluent UI Apple does  require resource bundles, we do not currently support Swift Package Manager.
+
+### Import and use the library
+
+After the framework has been added you can import the library to use it:
+
+For Swift
+```swift
+import FluentUI
+```
+For Objective-C
+```objective-c
+#import <FluentUI/FluentUI.h>
+```
+
+## Contributing
+
+Post bug reports, feature requests, and questions in [Issues](https://github.com/microsoft/fluentui-apple/issues).
```

Comment:
maybe a todo: should we consider adding guidelines about when creating a new control make sure it is added for both framework and static library target, and there should be an example in the demo/test app? maybe we should have a wiki page of todo checklist: accessibility/light/darkmode etc

Reply:
I wonder if that would make sense in a PR template? Like under a verification section, have a checklist of whether your change impacts Ax, dark mode, RTL, localization, etc and how you tested it?

Reply:
We can also have separate bug templates for bug reports and feature requests if we want. that information tracked at the issue level.